Web UI/CognitiveSearch.UI/wwwroot/js/results.js

```diff
@@ -319,12 +323,17 @@ function UpdateResults(data) {
             name = "";
         }
 
+        var thumbnail_size_ext = "_300xN.jpg";
```

Comment:
is this a suffix we're adding to each thumbnail image?
If so, is the correct place for this in the thumbnail generation code?